packages/opentelemetry-sdk-trace-base/test/common/Span.test.ts

```diff
@@ -147,7 +141,8 @@ describe('Span', () => {
       spanProcessor: tracer['_spanProcessor'],
     });
     span.end();
-    assert.ok(hrTimeToNanoseconds(span.duration) >= 0);
+    assert.ok(span.endTimeUnixNano != null);
+    assert.ok(span.endTimeUnixNano > span.startTimeUnixNano);
```

Comment:
looking at the browser tests, this seems to have the same timestamp.

Example from my run: 
`1741282349655000000 > 1741282349655000000`